prefect_dbt/cli/configs/base.py

```diff
@@ -26,6 +26,14 @@ class DbtConfigs(Block, abc.ABC):
             "but available in dbt."
         ),
     )
+    override_fields: bool = Field(
+        default=False,
+        description=(
+            "If True, will allow new keys to override existing keys, with keys "
+            "from TargetConfigs taking precedence, rather than raising an error "
+            "when there are duplicate keys."
+        ),
+    )
```

Comment:
I think it's more clear to name this `allow_field_overrides` since fields in `extras` are what are bing overridden.
```suggestion
    allow_field_overrides: bool = Field(
        default=False,
        description=(
            "If enabled, fields from dbt target configs will override fields provided in extras"
        ),
    )
```